server/middleware/rolloutMiddleware.ts

```diff
@@ -0,0 +1,29 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+import { anyPrisonInRollout, probationAreaInRollout } from '../utils/rolloutUtils'
+import logger from '../../logger'
+
+export default function rolloutMiddleware(): RequestHandler {
+  return async (req: Request, res: Response, next: NextFunction) => {
+    const { user } = res.locals
+
+    if (req.url === '/rollout-status') {
+      return next()
+    }
```

Comment:
Dont know if this `if` block is needed, since it will redirect there anyway .... or maybe it gets caught in a redirect loop if this block is not here?

Reply:
Yes - gets caught in a loop without it.